core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -463,8 +463,12 @@ public function compare($blnReturnBuffer=false)
 				$arrOrderFields = $objDcaExtractor->getOrderFields();
 
 				// Find the changed fields and highlight the changes
-				foreach ($to as $k=>$v)
+				foreach (array_keys(array_merge($to, $from)) as $k)
 				{
+					$deleted = !isset($to[$k]);
```

Comment:
I’m not sure if `null` values are possible here?
If they are, we should check for `array_key_exists()` I think.

```suggestion
					$deleted = !array_key_exists($k, $to);
```